app/js/main.js

```diff
@@ -955,3 +963,12 @@ function dev() {
 function reload() {
     win.reloadDev();
 }
+
+/**
+ * Get version number from package.json
+ */
+fs.readFile("package.json", "utf8", function (err, data) {
```

Comment:
If we are going to parse `package.json`, we are going to get more information from it then just the version. However, I think this is fine for now. :)